images/build/benv.sh

```diff
@@ -199,24 +195,6 @@ benv-resolve() {
     return 0
   fi
 
-  # Resolve npm versions
```

Comment:
So why we don't need npm benv anymore, could you check if its corresponding tests are also removed?

Reply:
This is because npm comes along with Node so we do not need to maintain this list. So if a user is expecting to use a particular npm version, they should just use corresponding Node version. This makes things simpler for us too.  Yes the tests were removed earlier but we missed cleaning up all this stuff.

Reply:
the way we set are npm is different now .. for each node_version we check for supporting npm version and set the npm version there so we dont use benv to set npm any more